src/test/regress/expected/background_rebalance_parallel.out

```diff
@@ -0,0 +1,176 @@
+SET citus.next_shard_id TO 85674000;
+SET citus.shard_replication_factor TO 1;
+ALTER SEQUENCE pg_dist_background_job_job_id_seq RESTART 17777;
+ALTER SEQUENCE pg_dist_background_task_task_id_seq RESTART 1460000;
+ALTER SYSTEM SET citus.background_task_queue_interval TO '1s';
+SELECT pg_reload_conf();
+ pg_reload_conf
+---------------------------------------------------------------------
+ t
+(1 row)
+
+SELECT 1 FROM citus_disable_node('localhost', :worker_2_port, synchronous:=true);
+ ?column?
+---------------------------------------------------------------------
+        1
+(1 row)
+
+/* Create two tables table1_coloc1, table2_coloc1 and in a colocation group  */
+CREATE TABLE table1_colg1 (a int PRIMARY KEY);
+SELECT create_distributed_table('table1_colg1', 'a', shard_count => 8, colocate_with => 'none');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE TABLE table2_colg1 (b int PRIMARY KEY);
+SELECT create_distributed_table('table2_colg1', 'b' , colocate_with => 'table1_colg1');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE TABLE table1_colg2 (a int PRIMARY KEY);
+SELECT create_distributed_table('table1_colg2 ', 'a', shard_count => 8, colocate_with => 'none');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE TABLE  table2_colg2 (b int primary key);
+SELECT create_distributed_table('table2_colg2', 'b' , colocate_with => 'table1_colg2');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+/* Activate a node so that we can rebalance */
+SELECT 1 FROM citus_activate_node('localhost', :worker_2_port);
+ ?column?
+---------------------------------------------------------------------
+        1
+(1 row)
+
+SELECT citus_set_node_property('localhost', :worker_2_port, 'shouldhaveshards', true);
+ citus_set_node_property
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT * FROM citus_rebalance_start();
+NOTICE:  Scheduled 8 moves as job xxx
+DETAIL:  Rebalance scheduled as background job
+HINT:  To monitor progress, run: SELECT * FROM citus_rebalance_status();
+ citus_rebalance_start
+---------------------------------------------------------------------
+                 17777
+(1 row)
+
+SELECT citus_rebalance_wait();
+ citus_rebalance_wait
+---------------------------------------------------------------------
+
+(1 row)
+
+/* Check that all the move tasks for shards of table1_colg1 and table2_colg1 sequentially take a dependency on each other.
+ * The all the move tasks of table1_colg2 and table2_colg2 sequentially take a dependency on each other *
+   The first move task scheduled in each group has no dependency.
+ */
+SELECT * FROM pg_dist_background_task_depend WHERE job_id = 17777 ORDER BY task_id ASC;
```

Comment:
The output of this query is hard to make sense of. It think it would help a lot if task_id and depends_on are both joined on pg_dist_background_task to get the actual command. 

Same for the second time this query is called below in this file. The output there currently is pretty much identical, even though it tests the reference table stuff.